examples/gnn_for_bench_dataset.ipynb

```diff
@@ -0,0 +1,272 @@
+      "Using backend: pytorch\n"
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Step 2: Loading Data.\n",
+    "\n",
+    "Next, we will finish loading the data and learn about the size of the Training and Testing datasets."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "source": [
+    "import os\n",
+    "from nn_meter.dataset import gnn_dataloader\n",
+    "\n",
+    "__user_dataset_folder__ = os.path.expanduser('~/.nn_meter/dataset')\n",
+    "target_device = \"cortexA76cpu_tflite21\"\n",
+    "\n",
+    "print(\"Processing Training Set.\")\n",
+    "train_set = gnn_dataloader.GNNDataset(__user_dataset_folder__, train=True, device=target_device) \n",
+    "print(\"Processing Testing Set.\")\n",
+    "test_set = gnn_dataloader.GNNDataset(__user_dataset_folder__, train=False, device=target_device)\n",
+    "\n",
+    "train_loader = gnn_dataloader.GNNDataloader(train_set, batchsize=1 , shuffle=True)\n",
+    "test_loader = gnn_dataloader.GNNDataloader(test_set, batchsize=1, shuffle=False)\n",
+    "print('Train Dataset Size:', len(train_set))\n",
+    "print('Testing Dataset Size:', len(test_set))\n",
+    "print('Attribute tensor shape:', next(train_loader)[1].ndata['h'].size(1))\n",
+    "ATTR_COUNT = next(train_loader)[1].ndata['h'].size(1)"
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Processing Training Set.\n",
+      "Processing Testing Set.\n",
+      "Train Dataset Size: 20732\n",
+      "Testing Dataset Size: 5173\n",
+      "Attribute tensor shape: 26\n"
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Step 3: Run and Test\n",
+    "\n",
+    "We can run the model and evaluate it now!"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "source": [
+    "if torch.cuda.is_available():\n",
+    "    print(\"Using CUDA.\")\n",
+    "# device = \"cpu\"\n",
+    "device = torch.device(\"cuda:0\" if torch.cuda.is_available() else \"cpu\")\n",
+    "\n",
+    "load_model = False\n",
+    "if load_model:\n",
+    "    model = GNN(ATTR_COUNT, 3, 400, 0.1).to(device)\n",
+    "    opt = torch.optim.AdamW(model.parameters(), lr=4e-4)\n",
+    "    checkpoint = torch.load('LatencyGNN.pt')\n",
+    "    model.load_state_dict(checkpoint['model_state_dict'])\n",
+    "    opt.load_state_dict(checkpoint['optimizer_state_dict'])\n",
+    "    # EPOCHS = checkpoint['epoch']\n",
+    "    EPOCHS = 0\n",
+    "    loss_func = checkpoint['loss']\n",
+    "else:\n",
+    "    model = GNN(ATTR_COUNT, 3, 400, 0.1).to(device)\n",
+    "    opt = torch.optim.AdamW(model.parameters(), lr=4e-4)\n",
+    "    EPOCHS=20\n",
+    "    loss_func = nn.L1Loss()\n",
+    "\n",
+    "lr_scheduler = CosineAnnealingLR(opt, T_max=EPOCHS)\n",
+    "loss_sum = 0\n",
+    "for epoch in range(EPOCHS):\n",
```

Comment:
Please add more comment to explain the code, such as "start training", "start validation", "save the best model", etc.